index.js

```diff
@@ -0,0 +1,20 @@
+//String, RegExp, DataTime
+function show1() {
+  let str = "ahb acb aeb aeeb adcb axeb".replace(/a.b/g, "+");
+  console.log(str);
+}
+show1();
+
+function show2() {
+  let str = "2+3 223 2223".replace(/2\+3/g, "!");
+  console.log(str);
+}
+show2();
+
+let date = new Date();
+let year = date.getFullYear();
+let month = date.getMonth();
+let day = date.getDay();
+console.log(year);
+console.log(month);
+console.log(day);
```

Comment:
I got following here:
```
2022     
3        
4  
```

but I expected today (14th April)
```
2022     
4        
14  
```

Please read about difference of `getDate()` and `getDay()` and features of `getMonth()` here: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date